Changelog.md

```diff
@@ -1,6 +1,8 @@
 ### 0.8.24 (unreleased)
 
 Language Features:
+ * Introduce global ``block.blobbasefee`` for retrieving the blob base fee of the current block.
+ * Yul: Introduce builtin ``blobbasefee()`` for retrieving the blob base fee of the current block.
```

Comment:
Cancun related changes should also be listed under [Target Options](https://docs.soliditylang.org/en/latest/using-the-compiler.html#target-options) in the docs.